README.md

```diff
@@ -88,9 +89,53 @@ to tools provided by the `reset`. The `obs_msgs` returned from the environment a
 general messages that are observations. The `reward` is a scalar value. The `done` is a boolean value. The `truncated`
 is a boolean value.
 
-Let's see a complete example for building an environment.
+## Functional Environments
 
-### Environment subclass and state
+The easiest way to create an environment is using the functional interface, which just uses functions and decorators to define environments. First, let's define what the environment looks like by defining its `start` function:
+
+```py
+from aviary import fenv
+
+
+@fenv.start()
+def my_env(topic):
+    return f"Write a story about {topic}", {"chosen_topic": topic}
```

Comment:
Imo it's intuitive the first tuple entry is a reset message, but the second tuple entry is not intuitive.

Can you add a comment or docs detailing this `dict` second value? Alternately, maybe we make the return be a `NamedTuple` you can import and use

Reply:
I'm assuming the second return is the env state right?

Reply:
Oh reading below, looks like it's `state.extras`